### PR TITLE
Ports testing script to JS to support all tests running on Windows

### PR DESCRIPTION
**Pull request by cfroystad**
Since I'm stuck on Windows for the time being (it's not all that bad 😄 ), I've ported the bash script to some form of JavaScript to be able to run the full test suite locally.

I'm creating the PR as an offer if it's useful to the project, but it will likely need some improvements by someone native to JavaScript - or at least some heavy code review. If we're happy with the current solution (bash is great after all) I have no problems just using this locally

### CODE REVIEW

**Comment by patrickt** [comment]
I certainly see no objection to using this script rather than bash. I’ll take a look at the code in a few.

File: package.json [hunk 1]
```diff
@@ -37,4 +37,4 @@
       "highlights": "queries/highlights.scm"
     }
   ]
-}
+}
```
Comment (patrickt) [inline package.json:40]: Can you preserve the newline here?

**Review by patrickt — APPROVED** [review]
Looks good to me modulo one nitpick.

File: package.json (hunk 1 above)
Comment (cfroystad) [reply package.json]: Good catch! I've updated my VS Code configuration not to remove them anymore

**Pull request merged**